caravel/storage/entities.py

```diff
@@ -55,6 +58,7 @@ def keywords(self):
         # Tokenize title and body (ranking them equally)
         words = self.title.split() + self.body.split()
```

Comment:
Er, sorry, maybe we're talking about separate things. I was thinking that since fold_query_term(...) operates on both emails and non-email terms, it might make sense to add the email to `words` in `.keywords` instead of adding it to singularized.